<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileTransItem</class>
 <widget class="QWidget" name="fileTransItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>86</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#fileTransItem { background: none }
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="PictureLabel" name="imageLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="nameLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sizeLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="styleSheet">
            <string notr="true">QProgressBar {
padding: 1px;
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
					stop: 0 #fff,
					stop: 0.4999 #eee,
					stop: 0.5 #ddd,
					stop: 1 #eee );
height: 6px;
}
</string>
           </property>
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="progressLabel">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="lowPanel" native="true">
     <property name="styleSheet">
      <string notr="true">QPushButton { color:blue;background:none;border:none;cusor:hand;           }</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button1">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>接受</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button2">
        <property name="text">
         <string>另存为</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button3">
        <property name="text">
         <string>拒绝</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>imageLabel</zorder>
  <zorder>nameLabel</zorder>
  <zorder>sizeLabel</zorder>
  <zorder>button2</zorder>
  <zorder>button3</zorder>
  <zorder>progressBar</zorder>
  <zorder>button1</zorder>
  <zorder>nameLabel</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PictureLabel</class>
   <extends>QLabel</extends>
   <header>PictureLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
